runtime/compiler/control/J9Options.cpp

```diff
@@ -1752,53 +1732,62 @@ J9::Options::fePreProcess(void * base)
          }
       else
          {
-            UDATA largePageSize = 0;
-            UDATA largePageFlags = 0;
-            // -Xlp<size>, attempt to use specified page size
-            if (lpSize > 0)
+
+         UDATA largePageSize = 0;
+         UDATA largePageFlags = 0;
+         int32_t lpArgIndex;
+         UDATA lpSize = 0;
+
+         // -Xlp<size>, attempt to use specified page size
+         if ((lpArgIndex=FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xlp", 0)) >= 0)
+            {
+            GET_MEMORY_VALUE(lpArgIndex, "-Xlp", lpSize);
+            }
```

Comment:
@gita-omr As noted previously, this is where I've moved the size argument extraction for `-Xlp<size>`.